libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSourceOptions.kt

```diff
@@ -44,6 +44,13 @@ class FollowingFrameOptions internal constructor() {
      */
     var maxZoom = 16.35
 
+    /**
+     * Focal point for following camera frames.
```

Comment:
```suggestion
     * Defines focal point for camera frames when in FOLLOWING mode with a non-zero pitch. The focal point remains centered when the camera pitch is 0 in FOLLOWING mode.
```

Reply:
Pitch has no direct impact on the calculation. Only the relationship with `maximizeViewableGeometryWhenPitchZero` is relevant, more on that in the other suggestion.